pkg/operator/helpers/helpers.go

```diff
@@ -790,6 +790,20 @@ func ConvertToFeatureGateFlags(component string, features []operatorapiv1.Featur
 	return flags, ""
 }
 
+func IsRegistrationFeatureGateEnabled(rc *operatorapiv1.RegistrationConfiguration, featureName featuregate.Feature) bool {
```

Comment:
The `ConvertToFeatureGateFlags` is conveient for registration and work, but the `IsRegistrationFeatureGateEnabled ` is used in the `**operator**` part.

I'm making the operator and registartion-agent using the same featureGate which is the MultipleHubs in the `ClusterManager.Spec.RegistrataionConfiguration.FeatureGates`.

Reply:
can we just set config.MultipleHubs = helpers.FeatureGateEnabled(...

and later use config.MultipleHubs in other places?

Reply:
The function is used by 2 controllers:
* the klusterlet_controller.go: used to render the manifests
* the klusterlet_ssar_controller.go:  used to skip bootstrapDegradedCondition when multiplehubs.

Where we set `config.MultipleHubs = helpers.FeatureGateEnabled` is in the klusterlet_controller.go, but the config here is a struct only used in render, I don't think it can also be used in the klusterlet_ssar_controller.

---

The `IsRegistrationFeatureGateEnabled ` is replace to `FeatureGateEnabled` to align with other usages.